<a href="https://colab.research.google.com/github/ro420898/Python-Math-LiMA-2023/blob/main/2023_08_23_potencias_perfectas_fibonacci.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

In [1]:
def es_potencia(n, pot):
    return int(n**(1/pot)) ** pot == n

def potencias_fibonacci(n, pot=2):
    if pot is None:
      pot = 2
    fibs = []
    a, b = 0, 1
    while a < n:
        if es_potencia(a, pot):
            fibs.append(a)
        a, b = b, a+b
    return fibs

potencias_fibonacci(10**20,2)

[0, 1, 1, 144]

En el artículo "Classical and modular approaches to exponential Diophantine equations I. Fibonacci and Lucas perfect powers", los autores Yann Bugeaud, Maurice Mignotte y Samir Siksek utilizan un enfoque combinado de técnicas clásicas y modulares para estudiar las potencias perfectas en la sucesión de Fibonacci.

En primer lugar, los autores dan nuevas mejoras a los límites para las formas lineales en tres logaritmos. Luego, aplican un enfoque modular basado en la teoría de curvas elípticas para demostrar que las únicas potencias perfectas en la sucesión de Fibonacci son 0,1,8 y 144.

Los resultados de este artículo confirman conjeturas que se habían planteado previamente, y proporcionan nuevos conocimientos sobre la estructura de las potencias perfectas en esta importante sucesión matemática.

En resumen, este artículo demuestra que las potencias perfectas en la sucesión de Fibonacci son muy raras. Las únicas potencias perfectas que aparecen en estas sucesiones son 0,1,8 y 144 en la sucesión de Fibonacci.